README.md

```diff
@@ -1,72 +1,96 @@
 
-CaImAn implements a set of essential methods required in the analysis pipeline of large scale calcium imaging data. Fast and scalable algorithms are implemented for motion correction, source extraction, spike deconvolution, and component registration across multiple days. It is suitable for both two-photon and one-photon fluorescence microscopy data, and can be run in both batch and online modes. CaImAn also contains some routines for the analysis of behavior from video cameras. A list of features as well as relevant references can be found [here](https://caiman.readthedocs.io/en/latest/CaImAn_features_and_references.html).
+# Quick start :rocket:
+To get started quickly working on a demo notebook, just take the following three steps:
 
-## Requirements
+### Step 1: Install caiman
+The following is all done in your anaconda prompt, starting in your base environment:
+  
+    conda install mamba -n base -c conda-forge   # install mamba in base environment
+    mamba create -n caiman -c conda-forge caiman # install caiman
+    conda activate caiman  # activate virtual environment
 
-Right now, CaImAn works and is supported on the following platforms:
-* Linux on 64-bit x86 CPUs
-* MacOS on 64-bit x86 CPUs
-* Windows on 64-bit x86 CPUs
+### Step 2: Download code samples and data sets
+Here we will create a working directory with code samples and related data in a folder called `caiman_data`. Once this step is done, you will easily be able to run any of the demos provided with Caiman. While still in the caiman virtual environment created in the previous step:  
 
-32G RAM is required for a good experience, and depending on datasets, more may be necessary.
+    caimanmanager.py install
 
-CaImAn presently targets Python 3.9. Parts of CaImAn are written in C++, but apart possibly during install, this is not visible to the user. There is also an [older implementation](https://github.com/flatironinstitute/CaImAn-MATLAB) of CaImAn in Matlab (unsupported). That version can be used with [MCMC spike inference](https://github.com/epnev/continuous_time_ca_sampler) 
+### Step 3: Try out a demo notebook
+In Step 2, you created a `caiman_data` folder in your home directory. Here, we `cd` to the demo notebooks folder in `caiman_data`, and fire up a jupyter notebook:
 
-### Other hardware
-* ARM-based versions of Apple hardware work (if on a 16G model), but currently happen under x86 emulation and we cannot support them as well. A native OSX port is planned for late 2021/early 2022.
-* Support for Linux on ARM (e.g. AWS Graviton) is not available (but it may work with the port of conda, if you compile Caiman yourself - we do not have binary packages and this is untested). If you care about this, please let us know.
+    cd home/caiman_data/demos/notebooks  # go to demo notebooks
```

Comment:
Remember that this is different on windows. Also, "home" is not something the user will be able to type and have work; needs to note somehow that it needs substitution (or use ~)

Reply:
It says "Note that what counts as `home` in the first command depends on your OS, so be sure to fill in your actual home directory."

Reply:
Right, but given that ~ works on OSX and Windows, and / doesn't work on Windows anyway, we may need to have separate OSX/Linux and Windows instructions for this. If we do, the OSX/Linux instructions can just say ~/caiman_data/demos/notebooks without people not-reading-carefully being tripped up, and the Windows users can have their `%HomeDrive%%HomePath%\caiman_data\demos\notebooks` which should just work (in theory, at least).